### PR TITLE
Cancel filter bug

### PR DESCRIPTION
**Pull request by LeonidArefev**

### CODE REVIEW

File: lib/src/utils/image_filter/image_filters.dart [hunk 1]
```diff
@@ -36,9 +36,10 @@ class ImageAppFilter {
   }
 
   Future<ImageFilterResult> setImage(img_tools.Image image) async {
-    _response_cache.mainImage = image; //Important to set it first before all
-    _response_cache.changedPart = null;
     await imgChannels.splitImage(image);
+    /// here ALFA channel will be removed from PNG or GIF images
+    /// For JPEG its not relevant
+    _response_cache = await getImage();
```
Comment (LeonidArefev) [inline lib/src/utils/image_filter/image_filters.dart:42]: this is a fix for **PIX-140**

File: lib/src/utils/image_filter/image_filters.dart [hunk 2]
```diff
@@ -61,6 +62,7 @@ class ImageAppFilter {
             ((imgChannels.sourceBlue[pointIndex]) & 0xff); //blue*/
       }
     }
+    imgChannels.resetSmallCacheAfterCancel();
```
Comment (LeonidArefev) [inline lib/src/utils/image_filter/image_filters.dart:65]: fix for **PIX-141**

**Review by pirminb97 — APPROVED** [review]

**Comment by pirminb97** [comment]
Works fine

**Comment by tkuenneth** [comment]
Once again, WE DO NOT MIX TICKETS!

**Pull request merged**